tests/driver_at86rf2xx/cmd.c

```diff
@@ -192,7 +192,7 @@ static size_t _parse_addr(uint8_t *out, size_t out_len, const char *in)
     const char *end_str = in;
     uint8_t *out_end = out;
     size_t count = 0;
-    int assert_cell = 1;
+    int expect_cell = 1;
```

Comment:
Accidental search and replace?

Reply:
Of course, I tried to replace as much as possible automatically. Obviously, the regexp was not good enough. Good catch.

Reply:
Dropped the commit.